lib/release/api.ts

```diff
@@ -10,13 +10,13 @@ const MAX_CONCURRENT_REQUESTS = 5;
 export interface Request {
 	/**
 	 * An instance of PineJS, appropriately authenticated and configured for the
-	 * API server to use. The compatible API versions are v5 and v6, so make sure to
+	 * API server to use. The compatible API versions are v5, v6, and v7, so make sure to
```

Comment:
I would not mention v5 & v6 any more, since the typings will complain.
A way to make it support both v5, v6 & v7 (and avoid the major) could be to also add the old types w/ a union, like
```
contract?: string | models.JsonType;
```
but for sure you need to double check that b/ both the current & new CLI that uses v7